com.course.zenkou/src/Lecture2/task2/Task2App.java

```diff
@@ -0,0 +1,61 @@
+package Lecture2.task2;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Task2App {
+
+    private static final Scanner scanner = new Scanner(System.in);
+    private static final Pattern regex = Pattern.compile("^[1-7]$");
+
+    public static void main(String[] args) {
+        String dayNumber;
+        do {
+            System.out.print("Enter day's week: ");
+            dayNumber = scanner.next();
+            if (isWeekDay(dayNumber)) {
+                scanner.close();
+                break;
+            } else {
+                System.out.println("You entered wrong day!");
+            }
+        } while (true);
+
+        switch (dayNumber) {
```

Comment:
good solution but you may read about  DateFormatSymbols().getWeekdays() from java 8